src/components/Sum.jsx

```diff
@@ -1 +1,20 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
+  <p>
+    Sum of
+    {` ${a} `}
+    and
+    {` ${b} `}
+    is
+    {` ${+a + +b}`}
```

Comment:
you can make it in one template string
```suggestion
    {`Sum of ${a} and ${b} is ${+a + +b}`}
```